styles2.css

```diff
@@ -8,6 +8,10 @@ header > img {
   display: none;
 }
 
+.mobile-menu {
+  display: none;
+}
+
```

Comment:
- [ ] [OPTIONAL] Hi @NabilHY, I have a suggestion about the menu as you can see below, the nav links overlap with the background content. In the video I instead suggest the use of opacity property so that we can make a distinction between the foreground and the background content.


https://user-images.githubusercontent.com/4247740/156063529-90a94718-522c-4c95-a77e-e4a9c2931820.mp4

Reply:
@Cathella I really appreciate your feedback, I will make the suggested changes in my next commit, Have a nice day.